worlds/shivers/__init__.py

```diff
@@ -202,6 +202,12 @@ def set_lobby_access_keys(items: Dict[str, int]):
             self.options.non_local_items.value |= {name for name, data in item_table.items() if
                                                    data.type in [ItemType.POT, ItemType.POT_COMPLETE]}
 
+        # Ixupi captures priority locations:
+        if self.options.ixupi_captures_priority == 1:
+            self.options.priority_locations.value |= (
+                {name for name in self.location_names if name.startswith('Ixupi Captured')}
+            )
```

Comment:
I know I said this on the other PR too, but these could use a constant instead of iterating every time. Could do one PR later that changes a lot of these.
Also, this whole change to `priority_locations` could be done a lot earlier, like in `generate_early`. That isn't strictly necessary but it would be nice if we ever wanted to move the priority/excluded resolution to be earlier on in generation.

Reply:
Yeah agree with the first bit, that's a future refactor (maybe I'll have time during my vacation to work on that, this laptop at least I will have with me)

In regards to the second part would then the bit above this (about the pot locations) also be good to go into generate early? I'll just move those now if that's the case.

Reply:
That would be great!